.github/workflows/smoke.yml

```diff
@@ -41,60 +41,74 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: dorny/paths-filter@v2
+      if: github.event_name != 'workflow_dispatch'
       id: changes
       with:
         filters: |
           actions:
             - Dockerfile
             - 'common/**'
+            - 'updater/**'
```

Comment:
I wonder if we could [re-use](https://stackoverflow.com/a/56171053) the shared filters:

```yaml
fruit:
  - &banana
    name: banana
    colour: yellow

food:
  - *banana
  - name: carrot
    colour: orange
```

Reply:
I was reading about this the other day and wondered if it was too weird to introduce. I'll give it a shot.

Reply:
Maybe good to do in its own PR?

Reply:
It's good here, already exposed a few copy paste errors!

Reply:
Actually it looks like this isn't possible with arrays, there's no way to flatten it, so it's placing the shared array in the outer array and I'm assuming that won't work well with this action.